.api-version

```diff
@@ -1 +1 @@
-5.2.0
+5.2.1
```

Comment:
Should this have been bumped as I thought we normally did this when the enterprise goes out? Either way it's only your fixes to the API docs, so this should be okay.

Reply:
I'm ok with the API docs updating with community ATM